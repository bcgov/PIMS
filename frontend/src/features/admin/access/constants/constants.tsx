import { ColumnWithProps } from 'components/Table';
import { AccessStatusDisplayMapper } from 'constants/accessStatus';
import * as React from 'react';
import { Link } from 'react-router-dom';
import { CellProps } from 'react-table';

import { RowActions } from '../components/RowActions';
import { IAccessRequestModel } from '../interfaces';

export const columnDefinitions: ColumnWithProps<IAccessRequestModel>[] = [
  {
    Header: 'IDIR/BCeID',
    accessor: 'username',
    align: 'left',
    Cell: (props: CellProps<IAccessRequestModel>) => {
      return (
        <Link to={`/admin/user/${props.row.original.userId}`}>{props.row.original.username}</Link>
      );
    },
  },
  {
    Header: 'First name',
    accessor: 'firstName',
    align: 'left',
    clickable: true,
  },
  {
    Header: 'Last name',
    accessor: 'lastName',
    align: 'left',
    clickable: true,
  },
  {
    Header: 'Email',
    accessor: 'email',
    align: 'left',
    clickable: true,
    minWidth: 200,
  },
  {
    Header: 'Position',
    accessor: 'position',
    align: 'left',
    clickable: true,
  },
  {
    Header: 'Status',
    accessor: 'status',
    align: 'left',
    clickable: true,
    width: 100,
    Cell: (props: CellProps<IAccessRequestModel>) =>
      AccessStatusDisplayMapper[props.row.original.status],
  },
  {
    Header: 'Agency',
    accessor: 'agency',
    align: 'left',
    clickable: true,
    minWidth: 200,
  },
  {
    Header: 'Role',
    accessor: 'role',
    align: 'left',
    clickable: true,
    minWidth: 200,
  },
  {
    Header: ' ',
    Cell: RowActions,
    width: 75,
  },
];
