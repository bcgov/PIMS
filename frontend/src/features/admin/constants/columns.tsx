import { ColumnWithProps } from 'components/Table';
import { IAgency, IAgencyRecord } from 'interfaces';
import * as React from 'react';
import { Link } from 'react-router-dom';
import { CellProps } from 'react-table';

export const columnDefinitions: ColumnWithProps<IAgencyRecord>[] = [
  {
    Header: 'Agency name',
    accessor: 'name',
    align: 'left',
    clickable: true,
  },
  {
    Header: 'Short name',
    accessor: 'code',
    align: 'left',
    clickable: true,
  },
  {
    Header: 'Description',
    accessor: 'description',
    align: 'left',
    clickable: true,
  },
  {
    Header: 'Parent Agency',
    accessor: 'parent',
    align: 'left',
    clickable: true,
    Cell: (props: CellProps<IAgency>) => {
      return (
        <Link to={`/admin/agency/${props.row.original.parentId}`}>{props.row.original.parent}</Link>
      );
    },
  },
];

export const adminAreasColumnDefinistions: ColumnWithProps<any>[] = [
  {
    Header: 'Area name',
    accessor: 'name',
    align: 'left',
    clickable: true,
  },
];
