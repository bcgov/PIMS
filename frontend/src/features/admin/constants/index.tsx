import * as React from 'react';
import { ColumnWithProps } from 'components/Table';
import { CellProps } from 'react-table';
import { Link } from 'react-router-dom';
import { IAgency, IAgencyRecord } from 'interfaces';

export const columnDefinitions: ColumnWithProps<IAgencyRecord>[] = [
  {
    Header: 'Agency name',
    accessor: 'name',
    align: 'left',
    sortable: true,
    clickable: true,
  },
  {
    Header: 'Short name',
    accessor: 'code',
    align: 'left',
    clickable: true,
    sortable: true,
  },
  {
    Header: 'Description',
    accessor: 'description',
    align: 'left',
    clickable: true,
    sortable: true,
  },
  {
    Header: 'Parent Agency',
    accessor: 'parent',
    align: 'left',
    clickable: true,
    Cell: (props: CellProps<IAgency>) => {
      return (
        <Link to={`/admin/agency/${props.row.original.parentId}`}>{props.row.original.parent}</Link>
      );
    },
    sortable: true,
  },
];
