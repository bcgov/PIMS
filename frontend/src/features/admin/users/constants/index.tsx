import { ColumnWithProps } from 'components/Table';
import * as React from 'react';
import { FaCheck, FaTimes } from 'react-icons/fa';
import { Link } from 'react-router-dom';
import { CellProps } from 'react-table';

import { RowActions } from '../components/RowActions';
import { IUserRecord } from '../interfaces/IUserRecord';

export const columnDefinitions: ColumnWithProps<IUserRecord>[] = [
  {
    Header: 'IDIR/BCeID',
    accessor: 'username',
    align: 'left',
    Cell: (props: CellProps<IUserRecord>) => {
      return <Link to={`/admin/user/${props.row.original.id}`}>{props.row.original.username}</Link>;
    },
    sortable: true,
  },
  {
    Header: 'First name',
    accessor: 'firstName',
    align: 'left',
    clickable: true,
    sortable: true,
  },
  {
    Header: 'Last name',
    accessor: 'lastName',
    align: 'left',
    clickable: true,
    sortable: true,
  },
  {
    Header: 'Email',
    accessor: 'email',
    align: 'left',
    clickable: true,
    sortable: true,
    minWidth: 200,
  },
  {
    Header: 'Position',
    accessor: 'position',
    align: 'left',
    clickable: true,
    sortable: true,
  },
  {
    Header: 'Active',
    accessor: 'isDisabled',
    align: 'left',
    clickable: true,
    sortable: true,
    width: 100,
    Cell: (props: CellProps<IUserRecord>) =>
      props.row.original.isDisabled ? <FaTimes /> : <FaCheck />,
  },
  {
    Header: 'Agency',
    accessor: 'agency',
    align: 'left',
    sortable: true,
    clickable: true,
    minWidth: 200,
  },
  {
    Header: 'Roles',
    accessor: 'roles',
    align: 'left',
    clickable: true,
    minWidth: 200,
  },
  {
    Header: 'Last Login',
    accessor: 'lastLogin',
    align: 'left',
    clickable: true,
    minWidth: 100,
  },
  {
    Header: 'Created On',
    accessor: 'createdOn',
    align: 'left',
    clickable: true,
    sortable: true,
    minWidth: 100,
  },
  {
    Header: ' ',
    Cell: RowActions,
    width: 75,
  },
];
