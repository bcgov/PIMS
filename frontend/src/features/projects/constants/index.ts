export * from './AgencyResponses';
export * from './DisposalWorkflows';
export * from './DisposeWorkflowStatus';
export * from './ProjectWorkflowComponents';
export * from './ReviewWorkflowStatus';
export * from './SPPApprovalTabs';
