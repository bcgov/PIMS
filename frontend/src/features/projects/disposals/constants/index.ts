export * from './defaultFilter';
export * from './defaultProjectForm';
