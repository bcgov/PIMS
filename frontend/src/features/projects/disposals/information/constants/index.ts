export * from './AddPropertyColumns';
export * from './PropertyColumns';
