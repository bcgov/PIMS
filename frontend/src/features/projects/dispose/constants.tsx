import { DisposeWorkflowStatus, ProjectWorkflowComponent } from '.';
import React from 'react';

//TODO: map to real components
export const projectWorkflowComponents: ProjectWorkflowComponent[] = [
  { component: () => <p>Step 1</p>, workflowStatus: DisposeWorkflowStatus.Draft },
  { component: () => <p>Step 2</p>, workflowStatus: DisposeWorkflowStatus.SelectProperties },
  { component: () => <p>Step 3</p>, workflowStatus: DisposeWorkflowStatus.UpdateInformation },
  { component: () => <p>Step 4</p>, workflowStatus: DisposeWorkflowStatus.RequiredDocumentation },
  { component: () => <p>Step 5</p>, workflowStatus: DisposeWorkflowStatus.Approval },
  { component: () => <p>Step 6</p>, workflowStatus: DisposeWorkflowStatus.Submitted },
];
