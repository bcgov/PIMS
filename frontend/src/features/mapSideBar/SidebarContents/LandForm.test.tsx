import React from 'react';
import noop from 'lodash/noop';
import { LandForm } from '.';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import { Router } from 'react-router-dom';
import { createMemoryHistory } from 'history';
import { useKeycloak } from '@react-keycloak/web';
import * as API from 'constants/API';
import { ILookupCode } from 'actions/lookupActions';
import * as reducerTypes from 'constants/reducerTypes';
import { fireEvent, render, wait } from '@testing-library/react';

const mockStore = configureMockStore([thunk]);
const history = createMemoryHistory();

const lCodes = {
  lookupCodes: [
    { name: 'agencyVal', id: '1', isDisabled: false, type: API.AGENCY_CODE_SET_NAME },
    { name: 'disabledAgency', id: '2', isDisabled: true, type: API.AGENCY_CODE_SET_NAME },
    { name: 'roleVal', id: '1', isDisabled: false, type: API.ROLE_CODE_SET_NAME },
    { name: 'disabledRole', id: '2', isDisabled: true, type: API.ROLE_CODE_SET_NAME },
  ] as ILookupCode[],
};

const store = mockStore({
  [reducerTypes.LOOKUP_CODE]: lCodes,
  [reducerTypes.PARCEL]: { parcels: [], draftParcels: [] },
});

const promise = Promise.resolve();

jest.mock('@react-keycloak/web');
(useKeycloak as jest.Mock).mockReturnValue({
  keycloak: {
    userInfo: {
      agencies: ['1'],
      roles: ['admin-properties'],
    },
    subject: 'test',
  },
});

const landForm = (
  <Provider store={store}>
    <Router history={history}>
      <LandForm
        handleGeocoderChanges={() => (promise as unknown) as Promise<void>}
        setMovingPinNameSpace={noop}
        handlePidChange={noop}
        handlePinChange={noop}
      />
    </Router>
  </Provider>
);

describe('Land Form', () => {
  it('component renders correctly', () => {
    const { container } = render(landForm);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('displays identification page on initial load', () => {
    const { getByText } = render(landForm);
    expect(getByText(/parcel identification/i)).toBeInTheDocument();
  });

  it('goes to corresponding steps', async () => {
    const { getByText, queryByText, getAllByText } = render(landForm);
    await wait(() => {
      fireEvent.click(getByText(/continue/i));
    });
    expect(getByText(/Strategic Real Estate Classification/i)).toBeInTheDocument();
    await wait(() => {
      fireEvent.click(getByText(/Continue/i));
    });
    expect(getAllByText(/Net Book Value/i)).toHaveLength(2);
    await wait(() => {
      fireEvent.click(getByText(/Continue/i));
    });
    expect(getByText(/Review your land info/i)).toBeInTheDocument();
    expect(queryByText(/continue/i)).toBeNull();
  });

  it('review has appropriate subforms', async () => {
    const { getByText } = render(landForm);
    await wait(() => {
      fireEvent.click(getByText(/Review/i));
    });
    expect(getByText(/parcel identification/i)).toBeInTheDocument();
    expect(getByText(/current zoning/i)).toBeInTheDocument();
    expect(getByText(/est'd market value/i)).toBeInTheDocument();
  });
});
