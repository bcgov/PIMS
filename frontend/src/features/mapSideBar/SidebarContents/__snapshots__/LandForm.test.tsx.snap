// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Land Form component renders correctly 1`] = `
"<div class=\\"sc-fzqBZW htlTcn landForm\\">
  <form action=\\"#\\" class=\\"sc-AxmLO haTFfH\\">
    <nav class=\\"hideTabs nav nav-tabs\\" role=\\"tablist\\"><a href=\\"#\\" role=\\"tab\\" data-rb-event-key=\\"0\\" id=\\"steppedform-tabs-tab-0\\" aria-controls=\\"steppedform-tabs-tabpane-0\\" aria-selected=\\"true\\" class=\\"nav-item nav-link active\\">
        <p>Tab 1</p><svg stroke=\\"currentColor\\" fill=\\"currentColor\\" stroke-width=\\"0\\" viewBox=\\"0 0 512 512\\" size=\\"15\\" height=\\"15\\" width=\\"15\\" xmlns=\\"http://www.w3.org/2000/svg\\">
          <path d=\\"M464 32H48C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h416c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48zm-83.6 290.5c4.8 4.8 4.8 12.6 0 17.4l-40.5 40.5c-4.8 4.8-12.6 4.8-17.4 0L256 313.3l-66.5 67.1c-4.8 4.8-12.6 4.8-17.4 0l-40.5-40.5c-4.8-4.8-4.8-12.6 0-17.4l67.1-66.5-67.1-66.5c-4.8-4.8-4.8-12.6 0-17.4l40.5-40.5c4.8-4.8 12.6-4.8 17.4 0l66.5 67.1 66.5-67.1c4.8-4.8 12.6-4.8 17.4 0l40.5 40.5c4.8 4.8 4.8 12.6 0 17.4L313.3 256l67.1 66.5z\\"></path>
        </svg>
      </a><a href=\\"#\\" role=\\"tab\\" data-rb-event-key=\\"\\" id=\\"steppedform-tabs-tab-\\" aria-controls=\\"steppedform-tabs-tabpane-\\" tabindex=\\"-1\\" aria-selected=\\"false\\" class=\\"nav-item nav-link\\"><button type=\\"button\\" class=\\"Button Button--icon-only primary btn btn-primary\\">
          <div class=\\"Button__icon\\"><svg stroke=\\"currentColor\\" fill=\\"currentColor\\" stroke-width=\\"0\\" viewBox=\\"0 0 448 512\\" size=\\"20\\" height=\\"20\\" width=\\"20\\" xmlns=\\"http://www.w3.org/2000/svg\\">
              <path d=\\"M416 208H272V64c0-17.67-14.33-32-32-32h-32c-17.67 0-32 14.33-32 32v144H32c-17.67 0-32 14.33-32 32v32c0 17.67 14.33 32 32 32h144v144c0 17.67 14.33 32 32 32h32c17.67 0 32-14.33 32-32V304h144c17.67 0 32-14.33 32-32v-32c0-17.67-14.33-32-32-32z\\"></path>
            </svg></div>
        </button></a></nav>
    <div class=\\"tab-content\\">
      <div id=\\"steppedform-tabs-tabpane-0\\" aria-labelledby=\\"steppedform-tabs-tab-0\\" role=\\"tabpanel\\" aria-hidden=\\"false\\" class=\\"fade tab-pane active show\\">
        <div class=\\"bs-stepper\\">
          <div class=\\"bs-stepper-header\\">
            <div class=\\"step active\\" data-target=\\"stepper-0\\"><button type=\\"button\\" class=\\"step-trigger\\"><span class=\\"bs-stepper-circle\\">1</span><span class=\\"bs-stepper-label\\">Parcel ID</span></button></div>
            <div class=\\"line\\"></div>
            <div class=\\"step\\" data-target=\\"stepper-1\\"><button type=\\"button\\" class=\\"step-trigger\\"><span class=\\"bs-stepper-circle\\">2</span><span class=\\"bs-stepper-label\\">Usage</span></button></div>
            <div class=\\"line\\"></div>
            <div class=\\"step\\" data-target=\\"stepper-2\\"><button type=\\"button\\" class=\\"step-trigger\\"><span class=\\"bs-stepper-circle\\">3</span><span class=\\"bs-stepper-label\\">Valuation</span></button></div>
            <div class=\\"line\\"></div>
            <div class=\\"step\\" data-target=\\"stepper-3\\"><button type=\\"button\\" class=\\"step-trigger\\"><span class=\\"bs-stepper-circle\\">4</span><span class=\\"bs-stepper-label\\">Review</span></button></div>
          </div>
        </div>
        <div class=\\"sc-fzqNJr cNbtvk\\">
          <div class=\\"sc-fzoyAV XHxrg\\">
            <div class=\\"parcel-identification\\">
              <div class=\\"container\\">
                <div class=\\"row\\">
                  <h4>Parcel Identification</h4>
                </div>
                <div class=\\"row no-gutters\\">
                  <div class=\\"col-md-6\\">
                    <div class=\\"d-inline-flex flex-nowrap form-row\\"><label class=\\"req form-label\\">*</label>PID<div class=\\"form-group\\"><input name=\\"pid\\" id=\\"input-pid\\" class=\\"input-small form-control\\" value=\\"\\"></div><label class=\\"form-label\\" style=\\"width: 35px; min-width: 35px; padding-left: 5px;\\">or<br>PIN&nbsp;</label>
                      <div class=\\"form-group\\"><input name=\\"pin\\" type=\\"number\\" id=\\"input-pin\\" class=\\"input-small form-control\\" value=\\"\\"><svg stroke=\\"currentColor\\" fill=\\"currentColor\\" stroke-width=\\"0\\" viewBox=\\"0 0 512 512\\" class=\\"tooltip-icon\\" height=\\"1em\\" width=\\"1em\\" xmlns=\\"http://www.w3.org/2000/svg\\">
                          <path d=\\"M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z\\"></path>
                        </svg></div>
                    </div>
                    <div class=\\"field-row row\\">
                      <p class=\\"label undefined\\">Street Address</p>
                      <div class=\\"GeocoderAutoComplete\\">
                        <div>
                          <div class=\\"form-group\\"><svg stroke=\\"currentColor\\" fill=\\"currentColor\\" stroke-width=\\"0\\" viewBox=\\"0 0 512 512\\" class=\\"tooltip-icon\\" height=\\"1em\\" width=\\"1em\\" xmlns=\\"http://www.w3.org/2000/svg\\">
                              <path d=\\"M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z\\"></path>
                            </svg><input name=\\"address.line1\\" field=\\"address.line1\\" id=\\"input-address.line1\\" class=\\"form-control\\" value=\\"\\"></div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div class=\\"col-md-1\\">
                    <h1>OR</h1>
                  </div>
                  <div class=\\"col-md-5\\">
                    <p>Click on the pin, and then click your desired location on the map to pull the parcel details.</p>
                    <div class=\\"row\\">
                      <div class=\\"marker-svg col\\"><button class=\\"sc-fznyAO hwQpCV\\"><svg class=\\"parcel-icon\\">draft-parcel-icon.svg</svg></button></div>
                    </div>
                  </div>
                </div>
                <div class=\\"row no-gutters\\">
                  <div class=\\"col-md-6\\">
                    <div class=\\"field-row row\\">
                      <p class=\\"label undefined\\"><span class=\\"req\\">*</span>Location</p>
                      <div class=\\"sc-AxjAm gLmrxD form-group\\">
                        <div class=\\"rbt\\" style=\\"outline: none; position: relative;\\" tabindex=\\"-1\\">
                          <div style=\\"display: flex; flex: 1; height: 100%; position: relative;\\"><input autocomplete=\\"off\\" type=\\"text\\" name=\\"address.administrativeArea\\" id=\\"address.administrativeArea-field\\" aria-autocomplete=\\"both\\" aria-expanded=\\"false\\" aria-haspopup=\\"listbox\\" role=\\"combobox\\" class=\\"rbt-input-main form-control rbt-input\\" value=\\"\\"><input aria-hidden=\\"true\\" class=\\"rbt-input-hint\\" readonly=\\"\\" style=\\"background-color: transparent; border-color: transparent; box-shadow: none; color: rgba(0, 0, 0, 0.35); left: 0px; pointer-events: none; position: absolute; top: 0px; width: 100%; border-style: inset inset inset inset; border-width: 2px 2px 2px 2px; line-height: normal; padding: 1px 1px 1px 1px;\\" tabindex=\\"-1\\" value=\\"\\"></div>
                        </div>
                      </div>
                    </div>
                    <div class=\\"field-row row\\">
                      <p class=\\"label undefined\\">Province</p>
                      <div class=\\"form-group\\"><select name=\\"address.provinceId\\" disabled=\\"\\" id=\\"input-address.provinceId\\" class=\\"form-select form-control\\">
                          <option value=\\"\\">Must Select One</option>
                        </select></div>
                    </div>
                    <div class=\\"postal row\\">
                      <p class=\\"label undefined\\">Postal Code</p>
                      <div class=\\"form-group\\"><input name=\\"address.postal\\" style=\\"width: 120px;\\" id=\\"input-address.postal\\" class=\\"input-small form-control\\" value=\\"\\"></div>
                    </div>
                    <div class=\\"field-row row\\">
                      <p class=\\"label undefined\\">Agency</p>
                      <div class=\\"AutoCompleteText\\">
                        <div class=\\"form-group\\">
                          <div class=\\"input-area\\"><input name=\\"agencyId\\" disabled=\\"\\" id=\\"input-agencyId\\" class=\\"form-control\\" value=\\"\\"></div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div class=\\"form-container col-md-6\\">
                    <div class=\\"field-row row\\">
                      <p class=\\"label undefined\\">Name</p>
                      <div class=\\"form-group\\"><input name=\\"name\\" id=\\"input-name\\" class=\\"form-control\\" value=\\"\\"></div>
                    </div>
                    <div class=\\"field-row row\\">
                      <p class=\\"label undefined\\">Description</p>
                      <div class=\\"form-group\\"><textarea name=\\"description\\" id=\\"input-description\\" class=\\"form-control\\"></textarea></div>
                    </div>
                    <div class=\\"field-row row\\">
                      <p class=\\"label undefined\\">Legal Description</p>
                      <div class=\\"form-group\\"><textarea name=\\"legalDescription\\" id=\\"input-legalDescription\\" class=\\"form-control\\"></textarea></div>
                    </div>
                    <div class=\\"field-row row\\">
                      <p class=\\"label undefined\\">Lot Size</p>
                      <div class=\\"form-group\\"><textarea name=\\"legalDescription\\" id=\\"input-legalDescription\\" class=\\"form-control\\"></textarea></div>
                    </div>
                  </div>
                  <div class=\\"col\\">
                    <div class=\\"input-medium harmful\\">
                      <p><span class=\\"req\\">*</span>Would this information be harmful if released?&nbsp;<a target=\\"_blank\\" rel=\\"noopener noreferrer\\" href=\\"https://www2.gov.bc.ca/gov/content/governments/services-for-government/policies-procedures/foippa-manual/disclosure-harmful-individual-public-safety\\">Policy</a></p>
                      <div class=\\"form-group\\">
                        <div class=\\"check-field\\">
                          <div class=\\"form-check\\"><input name=\\"data.isSensitive\\" type=\\"radio\\" id=\\"input-data.isSensitive\\" class=\\"form-check-input\\" value=\\"false\\"><label title=\\"\\" for=\\"input-data.isSensitive\\" class=\\"form-check-label\\">Yes</label></div>
                          <div class=\\"form-check\\"><input name=\\"data.isSensitive\\" type=\\"radio\\" id=\\"input-data.isSensitive-2\\" class=\\"form-check-input\\" value=\\"true\\" checked=\\"\\"><label title=\\"\\" for=\\"input-data.isSensitive-2\\" class=\\"form-check-label\\">No</label></div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div class=\\"sc-fzoLag ioFIqz\\">
            <div class=\\"sc-fzqAui dmUimR\\">
              <div class=\\"SresManual\\">
                <div><svg stroke=\\"currentColor\\" fill=\\"none\\" stroke-width=\\"2\\" viewBox=\\"0 0 24 24\\" stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" size=\\"28\\" height=\\"28\\" width=\\"28\\" xmlns=\\"http://www.w3.org/2000/svg\\">
                    <path d=\\"M2 3h6a4 4 0 0 1 4 4v14a3 3 0 0 0-3-3H2z\\"></path>
                    <path d=\\"M22 3h-6a4 4 0 0 0-4 4v14a3 3 0 0 1 3-3h7z\\"></path>
                  </svg></div>
              </div><small class=\\"sc-fzqMdD jntMyA p-1 mr-2\\">Inventory Policy</small>
            </div><span class=\\"sc-fzoXzr eLsvTE\\"></span><button type=\\"button\\" class=\\"btn btn-primary btn-sm\\">Continue</button>
          </div>
        </div>
      </div>
    </div>
    <div class=\\"container\\"></div>
  </form>
</div>"
`;
